File: Python/bee_1012.py
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

area_triangulo = (a * c) /2
area_do_circulo = 3.14159 * c**2
area_trapezio = (a + b) * c / 2
area_quadrado = b ** 2
area_retangulo = a * b

print(f"TRIANGULO: {area_triangulo:.3f}\nCIRCULO: {area_do_circulo:.3f}\nTRAPEZIO: {area_trapezio:.3f}\nQUADRADO: {area_quadrado:.3f}\nRETANGULO: {area_retangulo:.3f}")

File: C#/bee_1012.cs
using System;


class URI
{
    static void Main(String[] args)
    {
        String [] comprimentos = Console.ReadLine().Split(' ');
        double a = double.Parse(comprimentos[0]);
        double b = double.Parse(comprimentos[1]);
        double c = double.Parse(comprimentos[2]);

        double triangulo = (a *  c) / 2;
        double raio_do_circulo = 3.14159 * (c * c);
        double trapezio =(a + b) * c /2;
        double quadrado = b * b;
        double retangulo = a * b;

        Console.WriteLine($"TRIANGULO: {triangulo:F3}\nCIRCULO: {raio_do_circulo:F3}\nTRAPEZIO: {trapezio:F3}\nQUADRADO: {quadrado:F3}\nRETANGULO: {retangulo:F3}");

    }
}